tests/ha/single_pod_restart/test_data_pod_restart.py

```diff
@@ -964,70 +925,128 @@ def test_copy_obj_after_pod_restart(self):
-                    "copy object etags", self.object_name, self.bucket_name, bucket3)
+        bkt_obj_dict2 = bkt_obj_dict.copy()
+        t_t = int(perf_counter_ns())
+        if CMN_CFG["dtm0_disabled"]:
+            bkt_obj_dict.clear()
+            for cnt in range(bkt_cnt):
+                bkt_obj_dict[f"ha-bkt{cnt}-{t_t}"] = f"ha-obj{cnt}-{t_t}"
+        else:
+            for idx, bkt in enumerate(bkt_obj_dict):
+                bkt_obj_dict[bkt] = f"ha-obj{idx}-{t_t}"
+
+        LOGGER.info("Step 7: Perform copy object from already created/uploaded %s bucket to other "
+                    "buckets verify copy object etags", self.bucket_name)
         resp = self.ha_obj.create_bucket_copy_obj(event, s3_test_obj=s3_test_obj,
                                                   bucket_name=self.bucket_name,
                                                   object_name=self.object_name,
                                                   bkt_obj_dict=bkt_obj_dict,
                                                   put_etag=put_etag,
                                                   bkt_op=False)
         assert_utils.assert_true(resp[0], f"Failed buckets are: {resp[1]}")
-        LOGGER.info("Step 9: Performed copy of %s from already created/uploaded %s to %s and "
-                    "verified copy object etags", self.object_name, self.bucket_name, bucket3)
-
-        LOGGER.info("Step 10: Download the uploaded %s on %s & verify etags.", object3, bucket3)
-        resp = s3_test_obj.get_object(bucket=bucket3, key=object3)
-        LOGGER.info("Get object response: %s", resp)
-        get_etag = resp[1]["ETag"]
-        assert_utils.assert_equal(put_etag, get_etag, "Failed in verification of Put & Get Etag "
-                                                      f"for object {object3} of bucket {bucket3}.")
-        LOGGER.info("Step 10: Downloaded the uploaded %s on %s & verified etags.", object3, bucket3)
+        LOGGER.info("Step 7: Performed copy object from already created/uploaded %s bucket to other"
+                    " buckets verified copy object etags", self.bucket_name)
+
+        LOGGER.info("Step 8: Download the copied objects & verify etags.")
+        for bkt, obj in bkt_obj_dict2.items():
+            resp = s3_test_obj.get_object(bucket=bkt, key=obj)
+            LOGGER.info("Get object response: %s", resp)
+            get_etag = resp[1]["ETag"]
+            assert_utils.assert_equal(put_etag, get_etag, "Failed in verification of Put & Get "
+                                                          f"Etag for object {obj} of bucket "
+                                                          f"{bkt}.")
+        LOGGER.info("Step 8: Downloaded copied objects & verify etags.")
+
         LOGGER.info("COMPLETED: Test to verify copy object after data pod restart.")
 
-    # pylint: disable=C0321
     @pytest.mark.ha
     @pytest.mark.lc
     @pytest.mark.skip(reason="Blocked until F-22A is available")
```

Comment:
Do we need this skip marker now @sonalk0209

Reply:
no we can remove